addons/hr_holidays/report/holidays_summary_report.py

```diff
@@ -21,7 +21,7 @@ def _get_day(self, start_date):
         res = []
         start_date = fields.Date.from_string(start_date)
         for x in range(0, 60):
-            color = '#ababab' if start_date.strftime('%a') == 'Sat' or start_date.strftime('%a') == 'Sun' else ''
+            color = '#ababab' if start_date.weekday() in (5,6,) else '' # (5,6) = (Saturday, Sunday)
```

Comment:
Would also be nice to use `calendar.SATURDAY` and `calendar.SUNDAY` instead of random magic numbers.